interchaintest/two-party-pol/two_party_osmo_pol_test.go

```diff
@@ -0,0 +1,1821 @@
+		// 		require.Equal(t, int64(atomContributionAmount), atomBal)
+		// 		osmoBal, _ := cosmosOsmosis.GetBalance(ctx, partyBDepositAddress, nativeOsmoDenom)
+		// 		require.Equal(t, int64(osmoContributionAmount), osmoBal)
+		// 	})
+
+		// 	t.Run("tick until forwarders forward the funds to holder", func(t *testing.T) {
+		// 		for {
+		// 			holderOsmoBal := testCtx.queryNeutronDenomBalance(neutronOsmoIbcDenom, holderAddress)
+		// 			holderAtomBal := testCtx.queryNeutronDenomBalance(neutronAtomIbcDenom, holderAddress)
+		// 			holderState := testCtx.queryContractState(holderAddress)
+
+		// 			println("holder atom bal: ", holderAtomBal)
+		// 			println("holder osmo bal: ", holderOsmoBal)
+		// 			println("holder state: ", holderState)
+
+		// 			if holderAtomBal == atomContributionAmount && holderOsmoBal == osmoContributionAmount {
+		// 				println("holder/liquidpooler received atom & osmo")
+		// 				break
+		// 			} else if holderState == "active" {
+		// 				println("holderState: ", holderState)
+		// 				break
+		// 			} else {
+		// 				testCtx.tick(clockAddress, keyring.BackendTest, neutronUser.KeyName)
+		// 			}
+		// 		}
+		// 	})
+
+		// 	t.Run("tick until holder sends the funds to LPer and receives LP tokens in return", func(t *testing.T) {
+		// 		for {
+		// 			holderLpTokenBal := testCtx.queryLpTokenBalance(liquidityTokenAddress, holderAddress)
+		// 			println("holder lp token balance: ", holderLpTokenBal)
+
+		// 			if holderLpTokenBal == 0 {
+		// 				testCtx.tick(clockAddress, keyring.BackendTest, neutronUser.KeyName)
+		// 			} else {
+		// 				break
+		// 			}
+		// 		}
+		// 	})
+
+		// 	t.Run("lockup expires", func(t *testing.T) {
+		// 		for {
+		// 			testCtx.tick(clockAddress, keyring.BackendTest, neutronUser.KeyName)
+		// 			if testCtx.getNeutronHeight() >= uint64(expirationHeight) {
+		// 				break
+		// 			}
+		// 		}
+		// 	})
+
+		// 	t.Run("party A claims", func(t *testing.T) {
+		// 		for {
+		// 			routerAtomBalB := testCtx.queryNeutronDenomBalance(neutronAtomIbcDenom, partyBRouterAddress)
+		// 			routerOsmoBalB := testCtx.queryNeutronDenomBalance(neutronOsmoIbcDenom, partyBRouterAddress)
+		// 			routerAtomBalA := testCtx.queryNeutronDenomBalance(neutronAtomIbcDenom, partyARouterAddress)
+		// 			routerOsmoBalA := testCtx.queryNeutronDenomBalance(neutronOsmoIbcDenom, partyARouterAddress)
+
+		// 			println("routerAtomBalB: ", routerAtomBalB)
+		// 			println("routerOsmoBalB: ", routerOsmoBalB)
+		// 			println("routerAtomBalA: ", routerAtomBalA)
+		// 			println("routerOsmoBalA: ", routerOsmoBalA)
+
+		// 			if routerOsmoBalB != 0 {
+		// 				break
+		// 			} else {
+		// 				testCtx.tick(clockAddress, keyring.BackendTest, neutronUser.KeyName)
+		// 				testCtx.holderClaim(holderAddress, osmoNeutronAccount, keyring.BackendTest)
+		// 			}
+		// 		}
+
+		// 	})
+
+		// 	t.Run("tick routers until both parties receive their funds", func(t *testing.T) {
```

Comment:
Probably should remove the commented code in the test

Reply:
i will reuse those when integrating the new holder design in the follow up pr